plugins/userdata/ElectronUserData.cc

```diff
@@ -175,11 +187,13 @@ void ElectronUserData::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
 			const edm::Ptr<pat::Electron> elPtr(electrons, index);
 			std::vector < uint > idCutsToInvert { 99 };
 
+			// std::cout << "Energy Correction for electron : " << index << " is: " << electronSeedCorrections(iEvent, elPtr) << std::endl;
+			el.addUserFloat("energyCorrection", electronSeedCorrections(iEvent, elPtr));
```

Comment:
For jets we have
```c++
jett.addUserFloat("JECUncertainty", unc);
jet.addUserFloat("ptUp", ptUp);
jet.addUserFloat("ptDown", ptDown);
```
and
```c++
jet.addUserInt("isGood", passesIdAndEta && passesPt);
jet.addUserInt("isGoodPtDown", passesIdAndEta && passesPtDown);
jet.addUserInt("isGoodPtUp", passesIdAndEta && passesPtUp);
```

Since the correction also changes the electron p<sub>T</sub>, would it make sense to implement something like that for electrons as well?